<launch>
    
    <node name="landoltc3d_node" type="landoltc3d_node" pkg="pandora_vision_landoltc" respawn="False" />
    <!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->
    
    <rosparam file="$(find pandora_vision_landoltc)/cfg/landoltc_topics.yaml" 
        command="load" ns="landoltc3d"/>
        
    <rosparam file="$(find pandora_vision_landoltc)/cfg/landoltc_params.yaml" 
        command="load" ns="landoltc3d"/>
    
    <rosparam file="$(find pandora_vision_predator)/config/predator_params.yaml" 
        command="load" ns="landoltc3d"/>    
        
    <param name="patternPath" type="str" value="$(find pandora_vision_landoltc)/bold.jpg"/>  
           
</launch>
